blog/macos-rig/index.qmd

```diff
@@ -0,0 +1,309 @@
+
+```bash
+brew install zsh
+```
+
+5. Install [Oh My Zsh](https://ohmyz.sh/#install).
+
+Oh My Zsh (OMZ) is 'a delightful & open source framework for Zsh.' Or as ChatGPT succinctly put it:
+
+{{< tweet robbyrussell 1599596073282342914 >}}
+
+It comes with a ton of plugins, themes, and functions. For most users, having the great themes available from OMZ are enough to switch. But, the real value is [all the built-in OMZ plugins](https://github.com/ohmyzsh/ohmyzsh/wiki/Plugins) — which provide a rich set of aliases for Homebrew, Git, and many other command-line programs. For example, `gs` is `git status`, `bubo` is `brew update && brew outdated` (very helpful), etc. Here are some you should know:
+
+* The brew plugin and its aliases:[ https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/brew](https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/brew)
+* The git plugin:[ https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/git](https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/git)
+* The macos plugin for easier living through the Terminal: [https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/macos](https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/macos) 
+* There's even a plugin that reminds you what your aliases are! -- for maximum discoverability: [https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/aliases](https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/aliases)
+
+You can also hand-code your themes into your `.zshrc` dotfile instead of using the OMZ themes. The aliases not provided by OMZ can be added to the custom.zsh dotfile, such as the ones in the dotfiles linked below.
+
+Install OMZ with:
+
+```bash
+sh -c "$(curl -fsSL https://raw.github.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"
+```
+
+6. Configure [Oh My Zsh](https://github.com/gvelasq/dotfiles/tree/main/zsh).
+
+To configure oh-my-zsh, you can:
+
+1. **Edit your settings: **Navigate to your home directory `~/` and edit the `.zshrc` file.
+1. **Use preconfigured settings: **Navigate to your home directory `~/`. Replace the default `.zshrc` file with [this file](https://github.com/gvelasq/dotfiles/blob/main/zsh/.zshrc). Then, navigate to the hidden folder `~/.oh-my-zsh/custom` and drop [this file](https://github.com/gvelasq/dotfiles/blob/main/zsh/custom.zsh) in the folder.
+
+:::{.callout-tip}
+On a Mac, you can press Command + Shift + . to see your hidden folders.
+:::
+
+A `custom.zsh` file might be helpful because you can add many more commands for OMZ plugins than what are built in. For instance,
+
+```bash
+alias bog="brew outdated --greedy"
+```
+
+This allows you to use `bog` to run `brew outdated --greedy`, instead of having to type the whole command out. That particular command is helpful if you want to find ALL the programs that could potentially be upgraded.
+
+![](image1.jpg){fig-align="center" fig-alt="Keep Calm and bubo bog bupc bc" fig-width="400"}
+
+7. Go back to the Terminal and install [rig](https://github.com/r-lib/rig#linux).
```

Comment:
Right now the hyperlink points to the Linux installation section. Suggest to change the link to the macOS Homebrew section: https://github.com/r-lib/rig#macos-homebrew